src/style.css

```diff
@@ -1,3 +1,80 @@
+
 body {
   margin: 0;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+
+  --active-color: #00ACDC;
+}
+
+header{
```

Comment:
Not fixed 🤷 
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/c8de27e7-e5eb-49ec-a224-f1b40ca7011a)